### ISSUE TITLE
control size of figure output a la rmagic

### ISSUE THREAD

**Issue by shabbychef** [arokem/python-matlab-bridge#18]
The ```%R``` magic extension allows control of displayed figures in the iPython notebook, like so:
```
%R -w 700 -h 500
hist(rnorm(1000))
# etc
```
It should be easy to nick this code from the rmagic extension.

**Comment by blink1073** [comment]
@arokem, I can take this one.  Here's what I suggest:  add format and size to `make_figs` to make it look like this:

```matlab
function make_figs(figdir, fmt, width, height)
    figHandles = get(0, 'children');
    for fig=1:length(figHandles)
        f = figHandles(fig);
        p = get(f, 'position');
          if p(3) > width
                height = p(4) * width / p(3);
          end
          if p(4) > height
                width = p(3) * height / p(4);
          end
          size_fmt = sprintf('-S%d,%d', width, height);
          outfile = fullfile(figdir, ['MatlabFig', sprintf('%03d', fig)]);
          print_fmt = sprintf('-d%s', fmt)
          print(f, outfile, print_fmt, '-tight', size_fmt);
        close(fig);
    end;
endfunction;
```

And then mimic what [octave_magic](https://github.com/blink1073/oct2py/blob/master/oct2py/ipython/octavemagic.py#L189) uses for plot formatting.  This would allow us to provide this feature in the `matlab_kernel` (already provided in the `octave_kernel`).

**Comment by arokem** [comment]
+1 to all of that (and thanks!). I've noticed that new versions of matlab
the figures are much larger than they used to be, which is an annoyance.

**Comment by blink1073** [comment]
Closed by #134.
